### ISSUE TITLE
Review schema definitions for repetition of information from IDS/OCDS mapping

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/infrastructure#56]
From #34:

> Noting that (in-line with the previous definitions) the new definitions repeat some aspects of the OCDS input mapping, once the input mapping is merged into master we should review schema definitions and remove any repetitions.
